cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -62,14 +62,15 @@ func LogOrWriteConfig(fileName string, cfg *kubeschedulerconfig.KubeSchedulerCon
 		return fmt.Errorf("unable to locate encoder -- %q is not a supported media type", mediaType)
 	}
 
-	encoder := kubeschedulerscheme.Codecs.EncoderForVersion(info.Serializer, kubeschedulerconfigv1beta1.SchemeGroupVersion)
-	if klog.V(2).Enabled() {
-		bytes, err := runtime.Encode(encoder, cfg)
-		if err != nil {
-			return err
-		}
-		configString := string(bytes)
-		klog.Infof("Using component config:\n%+v\n", configString)
+	encoder := kubeschedulerscheme.Codecs.EncoderForVersion(info.Serializer, kubeschedulerconfigv1beta2.SchemeGroupVersion)
+
+	configFile, err := os.Create(fileName)
```

Comment:
wait... why is this implementation changing?

Reply:
This seemed to be a rebase issue. Fixed this.